main/src/library/Core/Io/File.flix

```diff
@@ -31,6 +31,51 @@ namespace Core/Io/File {
         import new java.io.File(String) as newFile;
         File(newFile(s))
 
+    ///
+    /// Combines every string in `n` into a single path
+    ///
+    pub def combine(n: Nel[String]): File & Impure = {
+        match n {
+            case Nel(base, xs) => List.fold((p, s) -> p.add(s), new(base), xs)
+        }
+    }
+
+    ///
+    /// Appends the string `s` to the end of the path `p`
+    ///
+    pub def add(f: File, s: String): File & Impure = {
```

Comment:
This we can call `resolve` then its fits better with the names used in the newer Java APIs.